### PR TITLE
Don't auto-populate products or location types

### PR DESCRIPTION
**Pull request by esoergel**
We decided to get rid of these.  Good idea to wait on build (although I ran it locally)
@benrudolph

### CODE REVIEW

**Comment by benrudolph** [comment]
build failure looks real

File: corehq/apps/commtrack/tests/util.py [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
-from corehq.apps.sms.backend import test
-from corehq.apps.commtrack.helpers import make_supply_point
 from corehq.apps.products.models import Product
-from couchforms.models import XFormInstance
-from dimagi.utils.couch.database import get_safe_write_kwargs
-from lxml import etree
+from corehq.apps.sms.backend import test
+from corehq.apps.users.models import CommCareUser
+
+from .. import const
```
Comment (benrudolph) [inline corehq/apps/commtrack/tests/util.py]: is `..` generally considered better than using the fully qualified path?

Comment (esoergel) [inline corehq/apps/commtrack/tests/util.py]: I really like the following pattern for imports:
```
# python standard library stuff

# django imports

# other 3rd party imports

# imports from other modules in this project

# explicit relative imports from this same module
```
With each block of imports alphabetized.  It's basically a progression from generic to specific.  Doing the explicit relative imports to me shows more clearly that these are imports from the same module - it's a way to see the cross-linking at a glance.  I won't always split these up quite as much, sometimes it'll be more like this:
```
# all 3rd party and python imports

# imports from other modules

# module imports
```
It depends on how many imports there are.  In this case, I know I'm gonna be doing a lot of commtrack work, so I arranged stuff to be less annoying.

**Comment by esoergel** [comment]
Should be g2m after build.

File: corehq/apps/commtrack/tests/util.py (hunk 1 above)
Comment (benrudolph) [inline corehq/apps/commtrack/tests/util.py]: nice i like that. the `..` does make it clearer that it is the same module. we should put this somewhere

**Comment by benrudolph** [comment]
:+1: i'll just wait for tests

**Comment by czue** [comment]
@agrzywinski  @kkrampa @lwyszomi fyi - don't think this impacts the migration code at all but in case it might you may want to know about it. we are no longer creating default stuff for new commtrack domains.

**Pull request merged**